packages/vitest-react-native/setup.js

```diff
@@ -75,6 +75,22 @@ const cacheExists = (cachePath) => fs.existsSync(cachePath)
 const readFromCache = (cachePath) => fs.readFileSync(cachePath, 'utf-8')
 const writeToCache = (cachePath, code) => fs.writeFileSync(cachePath, code)
 
+const processBinary = (code, filename) => {
+  const b64 = Buffer.from(code).toString('base64')
+  return `Buffer.from("${b64}")`
```

Comment:
Should it be `module.exports = Buffer.from("${b64}")`?

Reply:
> The way I understand it

No, this is not how it works. `addHook` returns the code that will be running when `require` is __called__. And `require` returns the result of `module.exports`. Please, refer to how CommonJS works in Node documentation: https://nodejs.org/api/modules.html#modules-commonjs-modules

Reply:
Got it 👍🏻  Added two commits now with `module.exports` and `, "base64"`